scripts/nesi_scripts/adv_im/clean_adv_im_tmps.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+if [[ $# -lt 2 ]];then
```

Comment:
I think that this should be part of "clean-up" step rather than as an individual script. (or as a separate adv-im cleanup step)